quiz.css

```diff
@@ -1,104 +1,101 @@
-body {
-	
-}
-
-
-
-	article {
-	/* display: block; */
-	border: 1px solid lightblue;
-	width: 30%;
-	height: 560.525px;
-	margin-left: 2.5%;
-	margin-top: 10px;
-	float: left;
-	/* background-color: #eee; */
-}
-/* .title { 
-}*/
-#pencil {
-	background-color: yellow;
-	text-align: center;
-	border: 1px dotted lightgray;
-	/* width: 100%; */
-	margin: 2px;
-}
-
-#inkpen {
-	background-color: orange;
-	text-align: center;
-	border: 1px dotted lightgray;
-	/* width: 100%; */
-	margin: 2px;
-}
-
-#sharpener {
-	background-color: greenyellow;
-	text-align: center;
-	border: 1px dotted lightgray;
-	margin: 2px;
-}
-
-#stapler {
-	background-color: red;
-	text-align: center;
-	border: 1px dotted lightgray;
-	margin: 2px;
-}
-
-#stamp {
-	background-color: gray;
-	text-align: center;
-	border: 1px dotted lightgray;
-	margin: 2px;
-}
-
-#notepad {
-	background-color: sandybrown;
-	text-align: center;
-	border: 1px dotted lightgray;
-	margin: 2px;
-}
-
-
-	a {
-	/* display: block; */
-}
-.justify {
-	text-align: justify;
-	padding: 0 15px;
-	}
-	.available {
-	font-weight: 600;
-	color: yellow;
-	background-color: darkgray;
-	padding-left: 20px;
-/* border: 2px solid #fff; */
-}
-img {
-	/* border: 2px solid #ddd; */
-	display: block;
-	height: 100px;
-	margin-left: auto;
-	margin-right: auto;
-	margin-top: 20px;
-	}
-	.specs {
-	text-align: center;
-	margin: 0;
-	/* padding: 0; */
-}
-.dim {
-	font-size: .8em;
-	line-height: .8em;
-	padding-left: 20px;
-	font-weight: bold;
-	}
-	footer {
-	font-size: .6em;
-	text-align: right;
-	padding-right: 10px;
-	}
-div {
-display: block;
+body {
+
+}
+
+article {
+	/* display: block; */
+	border: 1px solid lightblue;
+	width: 30%;
+	height: 560.525px;
+	margin-left: 2.5%;
+	margin-top: 10px;
+	float: left;
+	/* background-color: #eee; */
+}
+
+#pencil {
+	background-color: yellow;
+	text-align: center;
```

Comment:
Look how many times these three lines are repeated. How can you make this code DRYer?